test_scripts/pypeit_tests.py

```diff
@@ -465,51 +465,62 @@ class PypeItQuickLookTest(PypeItTest):
+        if self.setup.instr == 'keck_mosfire':
+            command_line = ['pypeit_ql_jfh_multislit', 'keck_mosfire']
             if self.pargs.quiet:
                 command_line += ['--no_gui', '--writefits']
         elif self.setup.instr == 'keck_lris_red_mark4':
-            command_line = ['pypeit_ql_multislit', 'keck_lris_red_mark4']
+            command_line = ['pypeit_ql_jfh_multislit', 'keck_lris_red_mark4']
             if self.pargs.quiet:
                 command_line += ['--no_gui', '--writefits']
-        elif self.setup.instr == 'keck_deimos':
-            # Two commands!
-            if self.command == 'calib':
-                command_line = ['pypeit_ql_keck_deimos', 
-                            f"{os.path.join(os.getenv('PYPEIT_DEV'), 'RAW_DATA', 'keck_deimos', 'QL')}", 
-                            "--root=DE.", "-d=3", 
-                            f"--redux_path={os.path.join(os.getenv('PYPEIT_DEV'), 'REDUX_OUT', 'keck_deimos', 'QL')}",
-                            "--calibs_only"]
-            elif self.command == 'science':
-                command_line = ['pypeit_ql_keck_deimos', 
-                            f"{os.path.join(os.getenv('PYPEIT_DEV'), 'RAW_DATA', 'keck_deimos', 'QL')}", 
-                            '--science=DE.20130409.20629.fits',  '--slit_spat=3:763',
-                            f"--redux_path={os.path.join(os.getenv('PYPEIT_DEV'), 'REDUX_OUT', 'keck_deimos', 'QL')}"]
-            else:
-                raise ValueError("Bad command")
         else:
-            command_line = ['pypeit_ql_mos', self.setup.instr]
-
-        if self.setup.instr != 'keck_deimos':
+            # Redux folder
+            redux_path = os.path.join(self.redux_dir,
+                    self.setup.instr, self.setup.name)
+            last_folder = 'QL'
+            if self.test_name is not None:
+                last_folder += '_' + self.test_name
+            redux_path = os.path.join(redux_path, last_folder)
+                    
+            command_line = [
+                'pypeit_ql', self.setup.instr,
+                '--redux_path', redux_path]
+
+        if self.setup.instr in ['keck_mosfire', 'keck_lris_red_mark4']:  # TESTING USING JFH QL
             command_line += [self.setup.rawdir] + self.files
-
+        else:
+            command_line += ['--full_rawpath', self.setup.rawdir, 
+                             '--rawfiles'] + self.files
+        # Aditional options
         for option in self.options:
             if self.options[option] is None:
                 command_line += [option]
+            elif self.options[option] in ['USE_MASTERS_DIR', 
+                                          'USE_CALIB_DIR']:
+                idir = os.path.join(self.redux_dir,
+                    self.setup.instr, self.setup.name)
+                # Masters?
+                if self.options[option] == 'USE_MASTERS_DIR':
+                    # Crazy hack for pypeit_setup 
```

Comment:
I think you can avoid this hack by using self.setup.rdxdir, which is updated in PypeItSetupTest to include the "instrument"_A directory.  You might have to run os.path.abspath on it though.

Reply:
nice! done